neural_compressor/adaptor/ox_utils/calibration.py

```diff
@@ -440,8 +440,9 @@ def get_weight_tensors_calib_range(self):
                 ),
             )
             calibrator = CALIBRATOR["minmax"]()  # use minmax method to calibrate initializer tensors
-            calibrator.collect(initializer_tensor)
-            weight_tensors_calib_range[initializer_tensor_name] = [list(calibrator.calib_range)]
+            if initializer_tensor.flatten().size > 0:
```

Comment:
do you know why the tensor size would be 0?

Reply:
initializer of some ops could be empty, for example Resize op:
<img width="105" alt="image" src="https://github.com/intel/neural-compressor/assets/65003808/7c6d96b9-2cda-420a-9589-6672265e604d">